views/participant.html.twig

```diff
@@ -457,6 +596,57 @@ $(document).ready(function() {
         height: 50,
         displayValue: true
     });
+
+    var orgPatientStatusData = {{ orgPatientStatusData|json_encode|raw }};
+
+    // Switch to default tab if empty
+    if (!orgPatientStatusData) {
+        $('[href="#patientstatus"]').tab('show');
+    }
+
+    // Switch to patient status tab if there is a form error
+    if ($('.patient-status-form').find('div').hasClass('alert-danger')) {
+        $('[href="#patientstatus"]').tab('show');
+        // Display form
+        setTimeout(function () {
```

Comment:
I remember you said you had some trouble with this. Is there a reason why you can't just call

```js
$('#patient-status-data-box').hide();
$('#patient-status-form-box').show();
```
?

Reply:
Yes, some how it didn't work when I call hide and show methods, so had to delay and trigger the click.